methods/getEmailContent.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const handlebars = require('handlebars');
+
+module.exports = function(template, data, allDone) {
```

Comment:
lets change this to templateName and read the base path from settings like the other method